### ISSUE TITLE
support default subtab in "more" sidebar tab, like for sidebar tab list

### ISSUE THREAD

**Issue by TheDiveO** [Jermolene/TiddlyWiki5#3127]
At this time, TW5 supports opening a default sidebar tab using `$:/config/DefaultSidebarTab`. At it seems from the sources, the "More" sidebar tab doesn't have the same feature. It would be helpful in some usecases, such as plugin development, if a TW5 instance could be configured, when loaded, to not only open the "More" sidebar tab, but also a certain subtab inside it. Something like `$:/config/DefaultSidebarMoreTab` would be needed.

**Comment by Jermolene** [comment]
Hi @TheDiveO good idea, I'd be happy to accept a PR if you or anyone else is interested in putting it together?

**Comment by TheDiveO** [comment]
Yes, I can do this. One question: I noted that `core/ui/SideBarLists` uses explicitly the macro call widget:
```
<$macrocall $name="tabs" tabsList=... default={{$:/config/DefaultSidebarTab}} state="$:/state/tab/sidebar" />
```
yet `core/ui/SideBar/More` uses macro call markup (implicit macro call widget):
```
<<tabs "..." "$:/core/ui/MoreSideBar/Tags" "$:/state/tab/moresidebar" "tc-vertical">>
```
Which style would you prefer?

**Comment by TheDiveO** [comment]
Ah, the magic of parameter substitution ... and text references in particular ;)

**Comment by TheDiveO** [comment]
merged with 29e4965, Döner.
